app/controllers/admin/api/cms/files_controller.rb

```diff
@@ -11,9 +11,12 @@ class Admin::Api::CMS::FilesController < Admin::Api::CMS::BaseController
 
   MAX_PER_PAGE = 100
   DEFAULT_PER_PAGE = 20
+  ALLOWED_PARAMS = %i[section_id path attachment downloadable tag_list].freeze
```

Comment:
I tried to test this by updating a File's tag_list but `acts_as_taggable` something something doesn't really want me to.
If we're expecting an array of values for `tag_list`, like
```json
{
  "file": [
    "best_tag_ever",
    "not_the_best_tag_ever"
  ]
}
```
I think we need to allow it with `tag_list: []`.
This came out of my not reliable memory so it could be wrong.

Reply:
The tags feature is not supported in the API endpoints. I created an issue ([THREESCALE-8996](https://issues.redhat.com/browse/THREESCALE-8996)) to add it but after a debate in slack we decided to not do it. I removed the tags from the file endpoints for consistency.